app/lib/fake_data/complaints.rb

```diff
@@ -0,0 +1,72 @@
+require "ffaker"
+
+module FakeData::Complaints
+  def self.complaint_data
+    {
+      id: FFaker::Random.rand(9999),
+      attributes: {
+        issueLastUpdated: FFaker::Time.date,
+        creationDate: FFaker::Time.date,
+        closedDate: FFaker::Time.date,
+        reopenedDate: FFaker::Time.date,
+        initialContactDate: FFaker::Time.date,
+        type: FFaker::Random.rand(2),
+        otherType: FFaker::Lorem.word,
+        issue: FFaker::Lorem.phrase,
+        priority: FFaker::Random.rand(3),
+        status: FFaker::Random.rand(3),
+        dueDate: FFaker::Time.date,
+        # these just sound cool for use as the grantee name :)
+        grantee: FFaker::AddressUK.street_name
+      },
+      relationships: {
+        grantAward: {
+          meta: {
+            id: FFaker::Random.rand(9999)
```

Comment:
[suggestion] This is less of a `must` than the top level `id` since it's in `meta` and non-standard, but I think we should use strings here and in the other relationships for the same reasons.

Reply:
Yeah, I think if we're making one id a string, they all ought to be strings